Compression/fcl/prolog.fcl

```diff
@@ -33,82 +36,20 @@ DetStepCompression : {
 }
 
 DigiCompressionTags : {
-    
     commonStrawDigiMCTag : "makeSD"
     commonExtraStepPointMCTags : [ "g4run:virtualdetector", "g4run:protonabsorber" ]
     commonCaloShowerSimTag : "CaloShowerROMaker"
     commonCaloShowerROTag : "CaloShowerROMaker"
     commonCrvDigiMCTag : "CrvWaveforms"
-
     primarySimParticleTag : "g4run"
     primaryCaloShowerStepTags : [ "CaloShowerStepMaker"]
 }
 
 DigiCompression : {
-  # redefine TimeMaps
     TimeMaps : [ "compressDigiMCs:protonTimeMap", "compressDigiMCs:muonTimeMap", "compressDigiMCs:cosmicTimeMap" ]
-    # for legacy data sets
-    NoCosmicTimeMaps : [ "compressDigiMCs:protonTimeMap", "compressDigiMCs:muonTimeMap" ]
-}
-
-RecoCompression : {
-    TimeMaps : [ "compressRecoMCs:protonTimeMap", "compressRecoMCs:muonTimeMap", "compressRecoMCs:cosmicTimeMap" ]
-    NoCosmicTimeMaps : [ "compressRecoMCs:protonTimeMap", "compressRecoMCs:muonTimeMap" ]
 }
-
 DigiCompression : {
   @table::DigiCompression
```

Comment:
I don't understand what this line is doing. Can the "Check" table not be included in the earlier DigiCompression table with TimeMaps?